api/types/user.go

```diff
@@ -423,6 +437,24 @@ func (u UserV2) GetGCPServiceAccounts() []string {
 	return u.getTrait(constants.TraitGCPServiceAccounts)
 }
 
+// GetHostUserUID retrieves the host user uid
+func (u UserV2) GetHostUserUID() string {
+	uid := u.getTrait(constants.TraitHostUserUID)
+	if len(uid) == 0 {
+		return ""
+	}
+	return uid[0]
+}
+
+// GetHostUserGID retrieves the host user uid
+func (u UserV2) GetHostUserGID() string {
+	gid := u.getTrait(constants.TraitHostUserGID)
+	if len(gid) == 0 {
+		return ""
+	}
+	return gid[0]
```

Comment:
A user can belong to multiple groups. Since the traits allow configuring multiple entries per trait name, only supporting the first one could be confusing to our users. Can we support all of them instead?

Reply:
I mention this in the comments in the RFD change, the plan is to only support using the first entry of the trait list, I meant to mention this in the docs

Reply:
Oh, sorry, didn't catch it. :)